pkg/apis/pipeline/v1/pipelinerun_types.go

```diff
@@ -283,6 +283,10 @@ type PipelineRunSpec struct {
 	// +optional
 	// +listType=atomic
 	TaskRunSpecs []PipelineTaskRunSpec `json:"taskRunSpecs,omitempty"`
+	// ManagedBy is the platform that is managing this PipelineRun.
+	// This field is immutable.
+	// +optional
```

Comment:
nit: can we add kubebuilder validation here which validates with the following regex 
```
"^[a-z0-9.-]+/[a-z0-9.-]+$"
```

Reply:
I am not sure whether it will cater to everything.